src/components/Home/VideoPlayerView.jsx

```diff
@@ -28,12 +62,19 @@ export const VideoPlayerView = ({ favorites }) => {
           allowFullScreen
           title="Embedded youtube"
         />
-        <button type="button">Add to favorites</button>
+        {isLogged && (
+          <Button onClick={() => handleAddFavorite(selectedVideo)} type="button">
+            {selectedVideo?.isFavorite ? 'Delete from favorites' : 'Add to favorites'}

```

Comment:
It would be good to create a `utils` file with a function like:
```

export const addOrDeleteFromFavorites(isFavorite) => (
  isFavorite  ? 'Delete from favorites' : 'Add to favorites'
)
```

In this way, you can call it from here and from other places where you are displaying this label, so if the label change in the future, you can change it in only one place 👍